nixos/modules/services/printing/cups-pdf.nix

```diff
@@ -0,0 +1,81 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.printing.cups-pdf;
+
+  reminder = ''
+    ### Don't edit this file. Set the NixOS option
+    ### ‘services.printing.cups-pdf.configurations’
+    ###
+    ### Read docs and find default config in
+    ### ${pkgs.cups-pdf}/share/doc
+
+
+  '';
+
+in
+
+{
+
+  options = {
+
+    services.printing.cups-pdf = {
```

Comment:
Would this be nicer as an option of the cups module? I think that would be easier to find.

Reply:
Seems not.  The option ```services.printing.cups-pdf``` is a subset of ```printing```, so it can be easily discoverable without polluting the cups module.